lib/shared/widgets/buttons/custom_green_elevated_button.dart

```diff
@@ -8,6 +12,10 @@ class CustomGreenElevatedButton extends StatelessWidget {
   final VoidCallback? onPressed;
   @override
   Widget build(BuildContext context) {
+    if(kDebugMode) {
+      final isFamilyApp = AppThemeSwitcher.of(context).isFamilyApp;
+      log('CustomGreenElevatedButton isFamilyApp: $isFamilyApp');
+    }
     return Padding(
```

Comment:
We already get logging when switching themes, so is this necessary? I guess no harm in it being there but just asking

Reply:
No not needed, will remove.